api/client.yaml

```diff
@@ -86,6 +86,92 @@ paths:
             application/json:
               schema:
                 $ref: 'https://raw.githubusercontent.com/moov-io/api/master/openapi-common.yaml#/components/schemas/Error'
+  /configuration/logo:
+    get:
+      tags: [Configuration]
+      summary: Retreive an organization's logo
+      description: Retrieve a previously-uploaded logo image from an organization configuration
```

Comment:
nit: 
I recommend removing `previously-uploaded` from this line.
`organization` might need to be possessive.

Reply:
I'll re-word it

Reply:
Let me know if the last commit makes more sense